packages/teams-js/test/public/webStorage.spec.ts

```diff
@@ -0,0 +1,81 @@
+import { app } from '../../src/public/app';
+import { FrameContexts, HostClientType } from '../../src/public/constants';
+import { _minRuntimeConfigToUninitialize } from '../../src/public/runtime';
+import { webStorage } from '../../src/public/webStorage';
+import { FramelessPostMocks } from '../framelessPostMocks';
+import { Utils } from '../utils';
+
+describe('webStorage', () => {
+  const mobilePlatformMock = new FramelessPostMocks();
```

Comment:
Would be clearer to the reader if you call this `framelessPlatformMock` and `framedPlatformMock` as per Shreyas' comment below -- they do not map 1-1 to frameless ==  mobile and utils == desktop.